lib/uninterleaveAndPackBuffers.js

```diff
@@ -98,7 +121,9 @@ function packAccessors(gltf, accessorsToPack, sourceBuffer, packBuffer, bufferVi
         var offset = byteOffset;
         var count = accessor.count;
         accessor.byteStride = 0;
-        accessor.bufferView = bufferViewId;
+        // Store as temp for now, finalize later
+        var extras = getPipelineExtras(accessor);
```

Comment:
This refers to the fact that it temporarily stores the new bufferViewId in extras and then permanently assigns it at the end to avoid naming conflicts. It isn't a TODO.